Add CI maintainer membership doc (#1488)
kubevirt/project-infra: docs/membership-policy.md

> # CI maintainers

Reviewer: This policy is inspired by k8s's and k6t's. Please link to the originals

Author: Yep, added links to both

Reviewer: I'm no github expert, but can you "Resolve" this?

This document outlines the various responsibilities of the maintainer role in
KubeVirt CI.

> KubeVirt CI maintainers are community members that have access to CI

Reviewer: ```suggestion
KubeVirt CI maintainers are community members that can control CI
```

infrastructure.

## Requirements

> * [Approvers] of https://github.com/kubevirt/project-infra for at least 1 month.

Reviewer: ```suggestion
* Be an [approver] of https://github.com/kubevirt/project-infra for at least 1 month.
```

> * Nominated by a CI maintainer.

Reviewer: ```suggestion
* Nominated by a current CI maintainer.
  - fgimenez
  - rmohr
  - dhiller
```

Reviewer: Or maybe list them elsewhere, in a machine-readable format?

Author: Yep, much better, I added an `OWNERS_ALIASES` file with a `ci-maintainer` alias according to https://www.kubernetes.dev/docs/guide/owners/#owners_aliases

## Responsibilities and privileges

> * Contribute infrastructure code.

Reviewer: please add examples

Author: Sure done.

> * Review and eventually approve infrastructure code proposed by others.

Reviewer: such as...

Author: Done,

> * Have access to the CI clusters, the infrastructure providers' UIs, the private

Reviewer: ```suggestion
* Have access to the CI clusters, user interface of the infrastructure providers, the private
```

> monitoring dashboards and the channels reserved to receive infrastructure

Reviewer: please name/link the channels

Author: Sure done.

# CI maintainers

This document outlines the various responsibilities of the maintainer role in
KubeVirt CI. It is inspired by the [KubeVirt community membership document] and
the [Kubernetes community membership document].

KubeVirt CI maintainers are community members that can control CI
infrastructure.

## Requirements

* Be an [approver] of https://github.com/kubevirt/project-infra for at least 1
month.

> * Nominated by a [current CI maintainer].

Reviewer: better English?
```suggestion
* A nomination by a [current CI maintainer].
```

> * A nomination by a [current CI maintainer].

Reviewer: ```suggestion
* A reasoned nomination by a [current CI maintainer].
```
Can we state where should the nomination take place? In kubevirt-dev mailing list? Just as a PR to the [current CI maintainer] list?

Author: I think it would be better just a PR with the changes to the list, and include the reasoning in the PR description, already pushed PTAL.

## Responsibilities and privileges

* Contribute infrastructure code, like Prow configuration, automation code,
create or modify manifests to deploy additional services or changes in the
monitoring stack.

> * Review and eventually approve infrastructure code proposed by others, such as
> in Prow configuration to add a new repo, new Prow job definitions or changes to
> tune existing alerts.

Reviewer: It was hard for me to parse the former sentence
```suggestion
* Review and eventually approve infrastructure code proposed by others. For example
Prow configuration to add a new repo, new Prow job definitions or
tuning existing alerts.
```
